test/binCases/help/help-output/__snapshots__/help-output.test.js.snap

```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`help-output 1`] = `
-"webpack-cli 3.1.2
+"webpack-cli 3.2.1
```

Comment:
I think we should remove the snapshot test.
Every time there's a release, it will break

Reply:
agree, but I think we'll have to revise this PR until the new cli is out, no way to look up dotfiles